src/main/java/io/cdap/plugin/sfmc/common/SalesforceMarketingCloudBaseConfig.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
```

Comment:
change year to 2023 for new files

Reply:
Changed it to 2023. Also, changed the year from 2022 to 2023 in new classes for MarketingConnector, MarketingConnectorConfig and MarketingConnectorTest.